analyses/2018_12_LABBsoft_tracking_review/2019_03_toktobaeva__literature_review_web_tracking/Flash cookies and Java JNLP PersistenceService.md

```diff
@@ -0,0 +1,73 @@
+# Flash cookies and Java JNLP PersistenceService
+### 1. Summary
+
+Adobe Flash stores two types of objects on users' computers: Local 
+Shared Objects (LOSes) and Remote Shared Objects (RSOs.) LOSes, while 
+generally referred to as Flash cookies, are fundamentally different 
+from HTTP cookies: store 100 Kb of data, harder to erase, cab be 
+accessed from all the browsers installed in the system from the same 
+storage and do not expire by default. RSOs allow for cooperating Flash 
+objects to access the contents locally. Unless passed to the third 
+party, the information on LOSes and RSOs can only be read by the domain 
+where Adobe Flash was used.
+
+Another way to store data is through Java JNLP Persistence system even 
+for applications in the untrusted execution environment. Data stored 
+with URLs. Applications are limited to only using those URLs. 
+
+### 2. Detection
+
+##### *In Literature*
+
+It is important to note that the detection accuracy relies on two 
+conditions: user not disallowing Flash cookies and not blocking Flash
+execution/Java applets. 
+
+Adobe supports ClearSiteData API (since v.10.3), which is invoked by all
+main browsers. The Flash manager allows for blocking of storage of any
+Flash objects.
+
+* Tracks in a private browsing?
+    * Yes
```

Comment:
For a specific claim like this it can be helpful to note which reference claimed it so people don't have to go digging. But this isn't a big deal.